### ISSUE TITLE
rediscloud_active_active_subscription data source should expose the regions for the subscription

### ISSUE THREAD

**Issue by atsui-bay1** [RedisLabs/terraform-provider-rediscloud#557]
It would be useful if the `rediscloud_active_active_subscription` data source exposed the `regions` for the subscription, as it is useful for suppling to the `rediscloud_acl_role` resource (and possibly others).

Note that it is currently possible to examine the `pricing` field to get this information, so this is mostly for convienence (`distinct(data.rediscloud_active_active_subscription.subscription.pricing.*.region)`)

### Affected Resource(s)
Please list the resources as a list, for example:
- rediscloud_active_active_subscription data source

**Comment by NoamSternRedis** [comment]
Hi @atsui-bay1, thank you for the feedback. 
We are adding a data source for the active_active_regions resource. So, this information will soon be possible through the new data source. Will it help you?

**Comment by atsui-bay1** [comment]
Yep, that will help clean up some of our terraform.  The workaround listed in the description is still effective, it just looks a bit weird.

**Comment by leonkatsevman** [comment]
Hi @atsui-bay1 , the version released on 22 of May has added the missing data source
please close the issue if resolved

`2.1.4 (22nd May 2025)
Added
Documentation for rediscloud_active_active_subscription_regions added.
Schema documentation amended to match documentation above.
`

**Comment by atsui-bay1** [comment]
Thank you for the update, I confirm that this new datasource works as expected
